docs/concepts-guidelines.md

```diff
@@ -308,10 +308,10 @@ The action can use GPG to sign commits with a GPG key that you generate yourself
 
       - uses: crazy-max/ghaction-import-gpg@v3
```

Comment:
These input names were actually changed in [v4 of that action](https://github.com/crazy-max/ghaction-import-gpg/releases/tag/v4.0.0), so lets bump the version in the docs, too.
```suggestion
      - uses: crazy-max/ghaction-import-gpg@v5
```